nilearn/regions/tests/test_region_extractor.py

```diff
@@ -51,10 +51,15 @@ def test_threshold_maps_ratio():
     # smoke test for function _threshold_maps_ratio with randomly
     # generated maps
 
-    # make sure that n_regions (4th dimension) are kept same even
-    # in thresholded image
+    # Check that the input maps are not changed
     maps, _ = generate_maps((6, 8, 10), n_regions=3)
+    maps.get_data()[:3] = 100
+    maps_data = maps.get_data().copy()
     thr_maps = _threshold_maps_ratio(maps, threshold=1.0)
```

Comment:
It would be good to have a small comment above: "test that there is no side effect".

Reply:
done

Reply:
Sorry, I hadn't seen your comment above :).